admin/static/js/pages/prereg-admin-page.js

```diff
@@ -0,0 +1,254 @@
+Notes.prototype.editItem = function() {
+    var self = this;
+    self.editing(true);
+    self.notes.edit(true);
+};
+
+Notes.prototype.stopEditing = function() {
+    var self = this;
+    self.editing(false);
+    self.notes.edit(false);
+};
+
+/**
+* The row containing the information for a single draft displayed in columns.
+*
+* @param  {Dict}    params     The draft information
+* @param  {String}  permission Whether the admin can assign a reviewer
+* @param  {List}    reviewers  The prereg reviewers who can be assigned
+*/
+var Row = function(params, permission, reviewers) {
+    var self = this;
+
+    self.params = params;
+    self.viewingDraft = ko.observable(false);
+    if (permission === "false") {
+        permission = false;
+    } else {
+        permission = true;
+    }
+    self.adminPermission = ko.observable(permission);
+
+    self.editing = ko.observable(false);
+
+    self.title = params.registration_metadata.q1.value || 'no title';
+    self.fullname = params.initiator.fullname;
+    self.username = params.initiator.emails[0].address;
+    self.initiated = self.formatTime(params.initiated);
+    self.updated = self.formatTime(params.updated);
+    self.status = ko.observable(params.is_pending_approval ? 'pending approval': 'approved');
+
+    self.proofOfPub = new ProofOfPub();
+    self.paymentSent = new PaymentSent();
+    self.notes = new Notes();
+    self.assignee = new Assignee(reviewers); 
+};
+
+
+Row.prototype.highlightRow = function(data, event) {  
+    var row = event.currentTarget;
+    $(row).css("background","#E0EBF3"); 
+};
+
+Row.prototype.unhighlightRow = function(data, event) {
+    var row = event.currentTarget;
+    $(row).css("background",""); 
+};
+
+Row.prototype.formatTime = function(time) {
+    var parsedTime = time.split(".");
+    return parsedTime[0]; 
+};
+
+Row.prototype.goToDraft = function(data, event) {
+    var self = this;
+    if (self.editing() === false) {
+        self.viewingDraft(true);
+        document.location.href = 'drafts/' + self.params.pk + '/';
+    }
+};
+
+/**
+* The view containing all of the rows with draft information.
+*
+* @param  {String}  adminSelector   The class to bind to
+* @param  {String}  user            The current user
+* @param  {List}    reviewers       The prereg reviewers who can be assigned
+*/
+var AdminView = function(adminSelector, user, reviewers) {
+    var self = this;
+
+    self.adminSelector = adminSelector;
+    self.user = user;
+    self.reviewers = reviewers;
+
+    self.getDrafts = $.getJSON.bind(null, "drafts/");
+
+    self.drafts = ko.observable([]);
+    self.loading = ko.observable(true);
+
+    self.sortBy = ko.observable('title');
+    self.sortedDrafts = ko.computed(function() {
+        var row = self.sortBy();
+        return self.drafts().sort(function (left, right) { 
+            var a = deep_value(left, row).toLowerCase();
+            var b = deep_value(right, row).toLowerCase();
+            return a == b ? 0 : 
```

Comment:
This could be `return a - b;`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort